### PR TITLE
build.gradle, Makefile: remove custom fdroid build

### PR DESCRIPTION
**Pull request by kari-ts**
We're no longer using GoogleSignIn, so there's no need for separate product flavors Updates tailscale/corp#18202

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -171,10 +171,6 @@ $(LIBTAILSCALE): Makefile android/libs $(LIBTAILSCALE_SOURCES) $(GOBIN)/gomobile
 
 libtailscale: $(LIBTAILSCALE)
 
-tailscale-new-fdroid.apk: $(LIBTAILSCALE)
-	(cd android && ./gradlew test assembleFdroidDebug)
-	mv android/build/outputs/apk/fdroid/debug/android-fdroid-debug.apk $@
-
 tailscale-new-debug.apk: $(LIBTAILSCALE)
 	(cd android && ./gradlew test assemblePlayDebug)
```
Comment (bradfitz) [inline Makefile:175]: but this says "PlayDebug" .... is it using any Play Store / Google APIs?

Comment (kari-ts) [reply Makefile]: nope, thanks for pointing this out!

**Comment by linsui** [comment]
The navigation-dynamic-features-fragment lib depends on gms. Maybe I should simply remove it in the fdroid build metadata?

**Comment by bradfitz** [comment]
@kari-ts, have you verified the build works on a non-Google-ified device?

**Comment by oxtoacart** [comment]
> navigation-dynamic-features-fragment

AFAICT, we're not actually using that library for anything, so we can just remove that dependency across the board.

**Review by oxtoacart — APPROVED** [review]

**Comment by kari-ts** [comment]
removed some additional unused dependencies.

verified the build mostly works on an AOSP emulator

**Pull request merged**